### ISSUE TITLE
Command Line parameters for batch processing

### ISSUE THREAD

**Issue by hgdagon** [elbereth/DragonUnPACKer#170]
I'm looking for the following options:
Input File
Input Directory
Output File
Output Directory
File Type (if known)
Straight to HyperRipper (ignore file type)
log (in-console)
log(file)

Currently the only command line options available are Language and File Open, so nothing can be done even with a batch file.

**Comment by elbereth** [comment]
Good idea. Now to re-install the dev tools on this new computer so I can do it... 💻
